src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeLibrary.cs

```diff
@@ -98,10 +100,21 @@ public static IntPtr Load(string libraryName, Assembly assembly, DllImportSearch
 
         /// <summary>
         /// NativeLibrary Loader: High-level API that doesn't throw.
+        /// Given a library name, this function searches specific paths based on the
```

Comment:
The matching update should also go into the official docs in https://github.com/dotnet/dotnet-api-docs. The xml comments are not synced up automatically.